substratevm/mx.substratevm/suite.py

```diff
@@ -642,6 +642,13 @@
                     "sun.util.locale",
                     "sun.invoke.util",
                 ],
+                "java.management": [
+                    "com.sun.jmx.mbeanserver",
```

Comment:
Do you only add those for the references in javadocs?

Reply:
Well the references help navigation, so I'd be for keeping them. It's just a bit unfortunate that we loose the reason in the `suite.py`. In there, it looks like a strong code dependency.

Reply:
I could add comments in `suite.py` explaining those imports are only needed for javadocs. Would this be better?

Reply:
Yes, that would be good. Thanks!